### PR TITLE
FIX: Include missing steps regarding how Dropbox is used to synchronize physio

### PR DESCRIPTION
**Pull request by esavary**
Partially addresses #400 . However, the information on setting up the synchronized folder on the AcqKnowledge computer is still missing.

### CODE REVIEW

File: docs/data-collection/scanning.md [hunk 1]
```diff
@@ -598,6 +598,8 @@ It is critical to check that physiological signals are looking good:
 - [ ] Check the box confirming that it has been acquired in the issue collecting notes about the session.
     Don't forget to report any observations there.
 
+- [ ] Copy data from the ./data subfolder in the GitHub repository HCPh-fMRI-tasks into the synchronized Dropbox folder, and make sure the Wi-Fi is on for synchronization.
```
Comment (oesteban) [inline docs/data-collection/scanning.md]: Did you have to do this manually in every session?

Comment (esavary) [reply docs/data-collection/scanning.md]: Yes, I did.

**Review by celprov — APPROVED** [review]

File: docs/data-collection/scanning.md (hunk 1 above)
Comment (celprov) [inline docs/data-collection/scanning.md]:
```suggestion
- [ ] Copy data from the `./data` subfolder in the GitHub repository HCPh-fMRI-tasks into the synchronized Dropbox folder, and make sure the Wi-Fi is on for synchronization.
```

Comment (celprov) [reply docs/data-collection/scanning.md]: Do Acknowledge and the Eye tracker save their data in the ./data subfolder in the HCPh-fMRI-tasks?
Can you also precise on which computer this happens?

**Review by celprov — CHANGES_REQUESTED** [review]

**Pull request merged**